package.json

```diff
@@ -5,8 +5,9 @@
   "main": "server/index.js",
   "scripts": {
     "test": "jest --forceExit",
+    "plough": "mysql -u root < schema.sql",
     "server-dev": "nodemon server/index.js",
-    "seed": "node project-utils/seed.js",
+    "seed": "npm run plough && node project-utils/seed.js",
     "build": "webpack --mode production",
     "react-dev": "webpack -d --watch",
     "start": "node server/index.js"
```

Comment:
✓ Nice! I think this will fix an error I was getting when trying to render Descriptions to my proxy server 👍🏽